scripts/eastern_kingdoms/scarlet_enclave/ebon_hold.cpp

```diff
@@ -694,6 +696,7 @@ struct npc_death_knight_initiate : public CreatureScript
                     // complete duel and evade (without home movemnet)
                     m_bIsDuelComplete = true;
                     m_creature->RemoveAllAurasOnEvade();
+                    m_creature->GetMap()->GetPlayer(m_duelerGuid)->RemoveAllAurasOnEvade();
```

Comment:
This can crash if the player is not found.

Reply:
Relocated to player check above.